tip/environments.py

```diff
@@ -0,0 +1,85 @@
+import os
+import json
+
+from tip.util import parse_package_specifier
+
+
+def locate(tip_dir: str, name: str) -> str:
+    """Find file of the environment called `name`."""
+    return os.path.join(_get_environments_dir(tip_dir), f"{name}.json")
+
+
+def read_environment_by_path(path: str) -> dict:
+    """Get package list of the environment at `path`."""
+    if not os.path.isfile(path):
+        raise FileNotFoundError(f"Environment file is not found at {path}")
+    with open(path, mode='r', encoding='utf8') as environment_file:
+        return json.load(environment_file)
+
+
+def read_environment_by_name(tip_dir: str, name: str) -> dict:
```

Comment:
Honestly such names doesn't make any sense for me. Read from where? From `tip_dir`? Os, but is `name` a filename? If so why not to specify just `path`? Wait a minute there are already `read_environment_by_path`… 🤯
Instead why not just call?
```python
load_environment(locate(env_name))
```